examples/src/operation/bootstrap/bootstrapPool.ts

```diff
@@ -0,0 +1,64 @@
+import { Bootstrap, poolUtils } from "@tinymanorg/tinyman-js-sdk";
+import { getAccount } from "../../util/account";
+import { getAssets } from "../../util/asset";
+import { algodClient } from "../../util/client";
+import signerWithSecretKey from "../../util/initiatorSigner";
+import { SDK_TEST_ARGS } from "../../util/other";
+
+/**
+ * Creates (i.e. "Bootstraps") a pool with an owned asset pair
+ */
+export async function bootstrapPool() {
+  const account = await getAccount();
+  const initiatorAddr = account.addr;
+  const { ids: assetIds } = await getAssets();
+  const [asset1ID, asset2ID] = assetIds;
+  const assetA = await algodClient.getAssetByID(asset1ID).do();
+  const assetB = await algodClient.getAssetByID(asset2ID).do();
+  const [asset_1, asset_2] = [assetA, assetB].map((asset) => ({
+    id: String(asset.index),
+    unit_name: asset.params["unit-name"],
+  }));
```

Comment:
Could just get the assets as arguments I guess to simplify the example a bit and to get straight to the point

```
bootstrapPool(asset1: {id: string; unit_name: string}, asset2: {id: string; unit_name: string})
```

Reply:
Yes agreed, it's repeated too much, these data don't change, so we can calculate once 👍